src/pollution_model/run_medusa_2.py

```diff
@@ -78,6 +80,29 @@ class MedusaRainfallEvent(NamedTuple):
     rainfall_ph: float
 
 
+class MedusaRainfallEventGeometry(NamedTuple):
+    """
+    Rainfall event parameters for MEDUSA 2.0 model.
+
+    Attributes
+    ----------
+    antecedent_dry_days: float
+        Length of antecedent dry period (days).
+    average_rain_intensity: float
+        Average rainfall intensity of the event (mm/h).
+    event_duration: float
+        Duration of the rainfall event (h).
+    rainfall_ph: float
+        The acidity level of the rainfall.
+    """
+
+    antecedent_dry_days: float
+    average_rain_intensity: float
+    event_duration: float
+    rainfall_ph: float
+    geometry: str
+
```

Comment:
This could be replaced with just using the `MedusaScenarios` table class, so you don't have to duplicate code